exercises/pangram/test/Tests.hs

```diff
@@ -0,0 +1,64 @@
+{-# LANGUAGE RecordWildCards #-}
+
+import Data.Foldable     (for_)
+import Test.Hspec        (Spec, describe, it, shouldBe)
+import Test.Hspec.Runner (configFastFail, defaultConfig, hspecWith)
+
+import Pangram (isPangram)
+
+main :: IO ()
+main = hspecWith defaultConfig {configFastFail = True} specs
+
+specs :: Spec
+specs = describe "pangram" $
+          describe "isPangram" $ for_ cases test
+  where
+    test Case{..} = it description $ isPangram input `shouldBe` expected
+
+-- Test cases adapted from
+-- `exercism/x-common/exercises/pangram/canonical-data.json`
+-- on 2017-03-28.
```

Comment:
I think at this point since there is a version at https://github.com/exercism/x-common/blob/master/exercises/pangram/canonical-data.json#L6 we should include it somehow, just not sure what format we want. "x-common version: 1.0.0" perhaps?